### PR TITLE
created wishlist-page

### PR DESCRIPTION
**Pull request by agrawalshanu708**
-created a wishlist page
-show all the wishlist products
Screenshot (89)

### CODE REVIEW

**Review by tanishk15sharma — APPROVED** [review]
good to merge

File: wishlist/wishlist.html [hunk 1]
```diff
@@ -0,0 +1,244 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (yashpatel1797) [inline wishlist/wishlist.html:6]: add author meta tag here.

File: wishlist/wishlist.css [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+}
+.Move-to-cart{
+    margin-top: 1rem;
+    width: 25rem;
+    height: 4rem;
+    font-size: 2rem;
+    font-family: 'Outfit', sans-serif;
+    border: none;
+    background-color: var(--primary-color) ;
+    color:black;
```
Comment (yashpatel1797) [inline wishlist/wishlist.css:53]: use variable here also.

**Review by yashpatel1797 — APPROVED** [review]
Looks good to merge.

File: wishlist/wishlist.css (hunk 2 above)
Comment (agrawalshanu708) [reply wishlist/wishlist.css:53]: ohkay

**Review by therajatg — COMMENTED** [review]
looks good to merge

**Pull request merged**